policy_sentry/querying/actions.py

```diff
@@ -161,50 +157,103 @@ def get_actions_at_access_level_that_support_wildcard_arns_only(
     return results
 
 
-def get_actions_with_access_level(service_prefix, access_level):
+def get_actions_with_arn_type_and_access_level(
+    service_prefix, resource_type_name, access_level
+):
     """
-    Get a list of actions in a service under different access levels.
+    Get a list of actions in a service under different access levels, specific to an ARN format.
+
+    Arguments:
+        service_prefix: A single AWS service prefix, like `s3` or `kms`
+        resource_type_name: The ARN type name, like `bucket` or `key`
+        access_level: Access level like "Read" or "List" or "Permissions management"
+    Return:
+        List: A list of actions that have that ARN type and Access level
+    """
+    service_prefix_data = get_service_prefix_data(service_prefix)
+    results = []
+
+    if resource_type_name == '*':
+        return get_actions_at_access_level_that_support_wildcard_arns_only(service_prefix, access_level)
```

Comment:
for resource_type_name == '*' calling the existing get_actions_at_access_level_that_support_wildcard_arns_only method